pre.js

```diff
@@ -29,5 +29,47 @@ var Module = {
         console.error(text);
     },
     canvas: {},
-    noInitialRun: true
+    noInitialRun: true,
+    onRuntimeInitialized: function() {
+        // Expose enum values.
+        var enums = {
```

Comment:
I had to move all this logic from `post.js` to `pre.js` inside the `onRuntimeInitialized` function. This is because these properties are not available yet in `post.js` since they are loaded later using `embind`, `onRuntimeInitialized` is triggered after loading `embind`. The interface still works the same because of this trick.